centaur/src/it/scala/centaur/reporting/BigQueryReporter.scala

```diff
@@ -153,7 +153,7 @@ class BigQueryReporter(override val params: ErrorReporterParams) extends ErrorRe
       "test_attempt" -> Option(testEnvironment.attempt + 1),
       "test_message" -> Option(centaurTestException.message),
       "test_name" -> Option(testEnvironment.name),
-      "test_stack_trace" -> centaurTestException.causeOption.map(ExceptionUtils.getStackTrace),
+      "test_stack_trace" -> ExceptionUtils.getStackTrace(centaurTestException),
```

Comment:
This should instead be:
```suggestion
      "test_stack_trace" -> Option(ExceptionUtils.getStackTrace(centaurTestException)),
```

Right now this is filtered out by the `collect` below`. In the corrected version, if the `getStackTrace` returns `null`, the stack trace won't be filled in.